web-api/terraform/modules/ci-cd/ci-cd-permissions.tf

```diff
@@ -357,19 +354,10 @@ resource "aws_iam_policy" "ci_cd_iam_policy" {
         "arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/migration_lambda_role_*",
         "arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/migration_status_lambda_role_*",
         "arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/reindex_status_lambda_role_*",
-        "arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/lambda_role_*",
```

Comment:
Do we not need the `lambda_role` any longer?

Reply:
We do not! It's been replaced by `switch_colors_cron_lambda_role_`